interfaces/builtin/dbus.go

```diff
@@ -184,6 +184,11 @@ dbus (send)
 dbus (receive, send)
     bus=###DBUS_BUS###
     peer=(name=###DBUS_NAME###, label=###SLOT_SECURITY_TAGS###),
+# allow connected snaps to ###DBUS_NAME### with a blank security label
+# to enable service activation
+dbus (receive, send)
+    bus=###DBUS_BUS###
+    peer=(name=###DBUS_NAME###, label="{,}"),
```

Comment:
I believe this could be combined with the previous line so there is one rule with `peer=(name=###DBUS_NAME###, label="{,###SLOT_SECURITY_TAGS###}"),`

That said, this continues to be annoying behavior from upstream that we never really cared for. This came up in https://github.com/snapcore/snapd/pull/4495 for snapd. In that PR, we added to the .service file:
```
AssumedAppArmorLabel=unconfined
```

In that light, I'd prefer instead of the above rule change that when snapd generates the service file for the snap, for it to create:

```
AssumedAppArmorLabel=snap.NAME.COMMAND
```

Reply:
I'd originally tried `label={,###SLOT_SECURITY_TAGS###}`, but that gets expanded to a quoted string, which doesn't fit into the glob.

I hadn't heard of `AssumedAppArmorLabel` before.  I've replaced these AppArmor changes with a cleaner version using that key, which seems to clear up the problems on 18.04+.  Thanks.